lib/src/settings.rs

```diff
@@ -208,6 +208,10 @@ impl UserSettings {
         self.config.get_string("revsets.log").unwrap_or_default()
     }
 
+    pub fn default_fix_revset(&self) -> String {
+        self.config.get_string("revsets.fix").unwrap_or_default()
+    }
```

Comment:
nit: better to inline in CLI code (and propagate error there)?

This also applies to `revsets.log`, but `settings.default_revset()` in lib tests would return `""`, which is invalid.

Reply:
Yeah, that makes sense. I was imitating the wrong parts of `revsets.log`.